packages/operator/pkg/trainer/trainer.go

```diff
@@ -254,6 +273,15 @@ func (mt *ModelTrainer) DownloadAlgorithm(k8sTraining *training.K8sTrainer) erro
 
 			return err
 		}
+		commit := &CommitInfo{
+			CommitID: commitID,
+		}
+		fileBytes, err := json.Marshal(commit)
+		if err != nil {
+			return err
+		}
+
+		_ = ioutil.WriteFile(commitInfoFile, fileBytes, 0644)
```

Comment:
For easier debugging in future, I'd check the returned error with logging if one occurs.